app/src/main/java/com/adiliqbl/wallpapers/util/Util.kt

```diff
@@ -1,15 +1,5 @@
 package com.adiliqbl.wallpapers.util
 
-import com.squareup.moshi.Moshi
-import com.squareup.moshi.Types
-import kotlin.reflect.KClass
-
-fun <T> getListFromJson(json: String, clazz: KClass<*>): MutableList<T> {
-    return Moshi.Builder().build()
-            .adapter<MutableList<T>>(Types.newParameterizedType(MutableList::class.java, clazz.java))
-            .fromJson(json)!!
-}
-
-fun toProperCase(s: String): String {
-    return s.substring(0, 1).toUpperCase() + s.substring(1).toLowerCase()
+fun String.properCase(): String {
+    return this.substring(0, 1).toUpperCase() + this.substring(1).toLowerCase()
```

Comment:
@adiliqbl The method is redundant as it is just capitalizing the string

Reply:
I don't think its possible to convert lowercase to sentence case purely with XML. One solution would be to override `setText` method of `TextView`, but I think is much cleaner than creating that.